### PR TITLE
have the magician tag a reviewer for external PRs

### PR DESCRIPTION
**Pull request by danawillow**
This was easier than trying to figure out the actual problem, and almost as good.

# Release Note for Downstream PRs (will be copied)
```releasenote

```

### CODE REVIEW

File: .ci/magic-modules/welcome-contributor.sh [hunk 1]
```diff
@@ -2,14 +2,20 @@
...
 cat > comment/pr_comment << EOF
 Hello!  I am a robot who works on Magic Modules PRs.
 
 I have detected that you are a community contributor, so your PR will be assigned to someone with a commit-bit on this repo for initial review.  They will authorize it to run through our CI pipeline, which will generate downstream PRs.
 
 Thanks for your contribution!  A human will be with you soon.
-EOF
 
-shuf -n 1 <(printf "danawillow\nrambleraptor\nchrisst\nrileykarson\nSirGitsalot\nslevenick\ntysen") > comment/assignee
+$ASSIGNEE, please review this PR or find an appropriate assignee.
```
Comment (rambleraptor) [inline .ci/magic-modules/welcome-contributor.sh]: Does there need to be an @ here for GitHub to recognize it as a user?

File: .ci/magic-modules/welcome-contributor.sh [hunk 2]
```diff
@@ -2,14 +2,20 @@
 
 set -x
 
+ASSIGNEE=$(shuf -n 1 <(printf "danawillow\nrambleraptor\nemilymye\nrileykarson\nSirGitsalot\nslevenick\ntysen"))
```
Comment (rambleraptor) [inline .ci/magic-modules/welcome-contributor.sh]: Not crazy about having this list here (we're bound to forget about it), but no use in putting in too much effort to make this perfect.

File: .ci/magic-modules/welcome-contributor.sh (hunk 1 above)
Comment (danawillow) [reply .ci/magic-modules/welcome-contributor.sh]: yes there does, thanks!

File: .ci/magic-modules/welcome-contributor.sh (hunk 2 above)
Comment (danawillow) [reply .ci/magic-modules/welcome-contributor.sh]: agreed, but don't have a better idea at the moment.

**Pull request merged**